src/sentry/incidents/endpoints/serializers.py

```diff
@@ -503,18 +519,30 @@ def validate(self, data):
             dataset = Dataset(data["dataset"].value)
             self._validate_time_window(dataset, data.get("time_window"))
 
+            conditions = copy(snuba_filter.conditions)
+            time_col = entity_subscription.time_col
+            conditions += [
+                [time_col, ">=", snuba_filter.start],
+                [time_col, "<", snuba_filter.end],
+            ]
+
+            body = {
+                "project": project_id[0].id,
+                "project_id": project_id[0].id,
```

Comment:
We probably don't, I just copied it blindly from `_create_in_snuba` . will remove

Reply:
Ok, I'm not totally sure on why both are there in `_create_in_snuba`, looks like some temporary thing from when we migrated to using `snuba_sdk`

Reply:
So for this, removing `project` fails the tests. seems like its necessary as per -> https://github.com/getsentry/snuba-sdk/blob/7cf587d46e41f5646c3353a5c58e73935b96baa1/snuba_sdk/legacy.py#L243
otherwise `project_id` does not get populated